### DISCUSSION TITLE
Built-in volume normalizer

### DISCUSSION THREAD

**Discussion by bcphung** [kyleneideck/BackgroundMusic#487] [Ideas]
As the title said, I can't really find any app for macOS to do the job.
Meanwhile, we have BackgroundMusic seems to be reasonable enough for that kind of feature.

**Comment by julian-weinert** [comment]
I would even improve on the idea and suggest a simple AU-Plugin interface.
So you can add whatever normaliser / compressor / limiter you want and it would definitely be simpler to implement even though for sure "slightly" more than a piece of cake
